awscli/customizations/utils.py

```diff
@@ -66,11 +66,30 @@ def rename_command(command_table, existing_name, new_name):
     del command_table[existing_name]
 
 
-def alias_command(command_table, existing_name, new_name):
-    """Moves an argument to a new name, keeping the old as a hidden alias."""
+def alias_command(command_table, existing_name, new_name, hide_current=True):
+    """Moves an argument to a new name, keeping the old as a hidden alias.
+
+    :type command_table: dict
+    :param command_table: The full command table for the CLI or a service.
+
+    :type existing_name: str
+    :param existing_name: The current name of the command.
+
+    :type new_name: str
+    :param new_name: The new name for the command.
+
+    :type hide_current: bool
```

Comment:
I'm debating if I like the ``hide_current`` parameter. While I am fine with the approach of reusing the ``alias_command`` and having a parameter that decides what is undocumented, I was a little confused at first before reading the docstring as I did not realize that by setting it to False, it would not hide the other command as well. I wonder if there is a better name to indicate this toggling side effect?